bin/ansible-runner-environment.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I think the request was to rename this file: `bin/ansible-runner-environment` (no `.sh`)